src/pages/restaurant/components/menu/components/dishes/Dishes.js

```diff
@@ -0,0 +1,55 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _map from 'lodash/map';
+import _startCase from 'lodash/startCase';
+import getKeysOfAnObject from '../../helpers/getKeysOfAnObject';
+
+//Components
+import CategoryDishes from './components/categoryDishes';
+
+//CSS
+import './dishes.css';
+
+const renderCategoryDishes = (dishes, rest) => (category) => {
+  return (
+    <CategoryDishes
+      key={category}
+      category={_startCase(category)}
+      categoryDishes={dishes[category]}
+      {...rest}
+    />
+  );
+};
+
+function Dishes(props) {
+  const { dishes, ...rest } = props;
+  const categories = getKeysOfAnObject(dishes);
```

Comment:
you can pass the categories as prop which were already present in parent component

Reply:
is it necessary to pass when we can get from here it self